### PR TITLE
Adding support for wholefoods, which uses standard schema.

### PR DESCRIPTION
**Pull request by ptindall**
Adding support for Whole Foods.

### CODE REVIEW

File: recipe_scrapers/wholefoods.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from ._abstract import AbstractScraper
+
+
+class WholeFoods(AbstractScraper):
+    @classmethod
+    def host(self, domain="com"):
+        return f"www.wholefoodsmarket.{domain}"
```
Comment (bfcarpio) [inline recipe_scrapers/wholefoods.py:7]: I'm not sure what we want to do for coding style. On one hand this makes it very simple, on the other there is some magic without the other functions explicitly defined as using the schema parser.

**Review by bfcarpio — CHANGES_REQUESTED** [review]

File: recipe_scrapers/__init__.py [hunk 2]
```diff
@@ -202,6 +203,7 @@
     Vegolosi.host(): Vegolosi,
     WatchWhatUEat.host(): WatchWhatUEat,
     WhatsGabyCooking.host(): WhatsGabyCooking,
+    WholeFoods.host(): WholeFoods,
```
[the remark this answers is not in the document]
Comment (ptindall) [reply recipe_scrapers/__init__.py:206]: Done.  I didn't realize they actually had stores in the UK.

**Review by bfcarpio — DISMISSED** [review]
Thanks!

File: recipe_scrapers/wholefoods.py (hunk 1 above)
Comment (ptindall) [reply recipe_scrapers/wholefoods.py:7]: I wasn't sure if there was a pattern for parsers that use the standard schema.

File: recipe_scrapers/wholefoods.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from ._abstract import AbstractScraper
+
+
+class WholeFoods(AbstractScraper):
```
Comment (bfcarpio) [inline recipe_scrapers/wholefoods.py:4]: Would you mind adding the other accessor methods to this? I previously thought we had a style preference, but couldn't find it. Now, I remembered that there was a commit (80fd5e3bc20062ffbf06386feb42924f4e16eac0) where we previously back filled in a lot of these methods.

**Review by bfcarpio — CHANGES_REQUESTED** [review]
Sorry to sound demanding! Just trying to keep everything up to the same style.

File: recipe_scrapers/wholefoods.py (hunk 3 above)
Comment (ptindall) [reply recipe_scrapers/wholefoods.py:4]: Added per provided commit.  Thank you.  I will follow this pattern in the future.

**Review by bfcarpio — APPROVED** [review]
Thank you for all your help and being quick to add the fixes. Hope you enjoy some nice recipes out of these.

**Pull request merged**